CHANGELOG.md

```diff
@@ -12,7 +12,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Bugfixes
 
-* None.
+* Fixed a bug that caused encrypted files created using
```

Comment:
I'd soften this somewhat since the bug only caused problems for a subset of files in specific conditions.